### PR TITLE
capstone project

### PR DESCRIPTION
**Pull request by cynthia-nwannah**
I built this project using HTML, CSS, and javascript. This is a fictional page built for my portfolio project
and is built based on an online website for a conference meeting and it was inspired by Cindy shin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To [Cindy Shin](https://www.behance.net/adagio07), since the whole website is based on her template.
+
```
Comment (ahmetbozaci) [inline README.md:55]: - [ ] Kindly add creative common license in the license section. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.)

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+- HTML & CSS
+- JavaScript
+
+## Live Demo
+Here is how it looks [live](https://cynthia-nwannah.github.io/capstone-project/).
+
+## Loom Video
+I have presented the project in a [Loom Video](https://www.loom.com/share/721a171dca5249509f6ba105630a5a3b).
+
```
Comment (ahmetbozaci) [inline README.md]: - [ ] Kindly the record video again. You are showing the only page. You should also show some codes that you are proud of it. Also, you can show more of the website. Video is just 1.20 min [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=A%20description%20of,very%20proud%20of.)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+               <li class ="list"> <a href="index.html">Home</li></a>   
+               <li class ="list"> <a href="about.html">About</li></a>   
+               <li class ="list"><a href="#">Program</li></a>
+               <li class ="list"><a href="#">join</li></a>
+               <li class ="list"><a href="#">sponsor</li></a>
+               <li class ="list"><a href="#">News</li></a>
+               <li class ="list"><a href="#" class="liste">CC campaign</li></a>
+            </ul>
+        </nav>
+    </header>
```
Comment (ahmetbozaci) [inline index.html]: **To follow design kindly fix these requirement:** [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)
- [ ] You missed the black navbar. Kindly add it. 
- [ ] Navbar has no home link in the original design. Kindly delete it.
- [ ] Font family is different from the original design. Please change it.
- [ ] CC campaign border is thicker than the original design. Kindly change it.
image
___
### ORIGINAL

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                        <i class="fa-solid fa-people-group"></i>
+                        </div>
+                    <h2 class="title">Workshop</h2>
+                    <p>Try creating your own work using open license</p>
+                </div>
+            </div>
+
+            <a href="#" class="promore">SEE THE WHOLE PROGRAM</a>
+            <button type="button" class="probtn">Join Microverse</button>
+        </section>
```
Comment (ahmetbozaci) [inline index.html]: ### MOBİLE VERSION

- [ ] It has some space between the program section and the main section. Kindly fix it.


image
____
### ORIGINAL
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            <div class="partner-logo">
+                <img src="asset/butter-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Conspicuous-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/fitbit-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Muscle-Lab-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/technogym-1-removebg-preview.png" alt="">
+            
+            </div>
+        </div>
+
```
Comment (ahmetbozaci) [inline index.html]: - [ ] Background images are different and smaller. Kindly change it.
- [ ] All section has left and right space in the original. Kİndly fix it.

image

___
### ORIGINAL
image

____
### MOBİLE VERSİON
### OPTIONAL
- [ ] It should have 2 speakers with more button.
image
____
### ORIGINAL
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+                     </div>
+                    </div>
+                <div class=" two">
+                    <h3>2018</h3>
+                    <p>internation submit in india</p>
+                </div>
+            </div>
+          </section>
+    
+    </main>
```
Comment (ahmetbozaci) [inline about.html]: - [ ] Pİctures are smaller in the original kindly change it.
- [ ] Font family is different. Kindly change it to all pages.
- [ ] `See The Past Fitness Submit title has a red horizontal line. Kindly add it
image
___
 ### ORIGINAL

image
___
### MOBİLE VERSİON
- [ ] Texts on the picture should be smaller. Kindly change it.

image

____
### ORIGINAL
image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+            <div class="partner-logo">
+                <img src="asset/butter-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Conspicuous-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/fitbit-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Muscle-Lab-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/technogym-1-removebg-preview.png" alt="">
+            
+            </div>
+        </div>
+    </footer>
```
Comment (ahmetbozaci) [inline about.html]: ### MOBİLE VERSİON
- [ ] Footer has two sections in the original kindly add it.
- [ ] Also partner section icons moving to the next line. Kindly change it.

image


### ORIGINAL
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @cynthia-nwannah  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You deploy the project and added it to the readme file.✔️
- You created speakers with javascript.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly don't work on the project after sending the review. If you fix something and ı make it a requirement kindly ignore it.
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html (hunk 4 above)
Comment (cynthia-nwannah) [reply index.html]: > * [ ]  The background image is different from the original kindly change it.
> 
> 
> image
> ### ORIGINAL
> 
> image
> ### MOBİLE
> 
>     * [ ]  It has some space between the program section and the main section. Kindly fix it.
> 
> 
> image ORIGINAL image

sorry I cant get the background image that is why I use the color

File: index.html (hunk 5 above)
Comment (cynthia-nwannah) [reply index.html]: @ahmetbozaci  this is optional and it is not required

File: index.html (hunk 4 above)
Comment (cynthia-nwannah) [reply index.html]: and it is not aa must you use the same image

File: index.html (hunk 5 above)
Comment (ahmetbozaci) [reply index.html]: @cynthia-nwannah You are right. You don't have to implement more buttons. But please at least make more good background images under the speakers. İmage can be different but the layout should be the same.

File: index.html (hunk 4 above)
Comment (ahmetbozaci) [reply index.html]: @cynthia-nwannah You are right to keep the image.

File: about.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About page</title>
+ <link rel="stylesheet" href="about.css">
+ <script src="https://kit.fontawesome.com/2849147338.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
```
Comment (elyor-doniyorov) [inline about.html:12]: - [ ] You added a` black navbar` to only home page but if you take a look at the below screenshot of the original design you can see it is in the about page as well. Kindly add it as the previous reviewer requested👇 
- [ ] Also, in your about page there is still a `home` link. Kindly remove it instead you can use your logo as a link to your home page. 👇 
- Original design :
Screenshot (275)
- Your design :
Screenshot (276)

File: index.html [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+               <p class="fit">To be physically fit means to be in a state of health and well-being. 
+                Physical fitness is defined as the body's ability to function efficiently and effectively in work and leisure activities, to be healthy, to resist disease and to react to emergency situations.</p>
+                <p class="date">Fri 26th - Sun 28th August Bank Holiday Weekend <br>
+               <span> @raffles city convention center, singapore and more</span></p>
+             </div>
+            </div>
+          </div>
+        </section>
+        <section class="main-bottom">
+            <h2 class="section-title" id="main-program">Main Program</h2>
```
Comment (elyor-doniyorov) [inline index.html:61]: - [ ] You have not implemented requested change by the previous reviewer which was about background image of main program section. Kindly implement that.

**Comment by cynthia-nwannah** [comment]
Thank you

Sent from Yahoo Mail on Android 
 
  On Fri, Jun 24, 2022 at 9:11 AM, Ahmet ***@***.***> wrote:   


@ahmetbozaci commented on this pull request.

In index.html:
> +        <section class="main-bottom">
+            <h2 class="section-title">Main Program</h2>
+            <div class="main-bottom-in">
+                <div class="card">
+                    <div class="pro">
+                    <i class="fa-solid fa-address-card"></i>
+                    </div>
+                    <h2 class="title">Lectures</h2>
+                    <p>listen to the speakers from various countries talking about fitness</p>
+                 </div>
+                <div class="card">
+                    <div class="pro">
+                    <i class="fa-solid fa-phone-flip"></i>
+                    </div>
+                    <h2 class="title">caller</h2>
+                    <p>Meet your linda orls and talk to her on which topics you are interested to</p>
+                </div>
+                <div class="card">
+                    <div class="pro">
+                        <i class="fa-solid fa-message"></i>
+                        </div>
+                    <h2 class="title">Forums</h2>
+                    <p>Share your thoughts and opinions with other fitness mentor. </p>
+                </div>
+                <div class="card">
+                    <div class="pro">
+                        <i class="fa-solid fa-clock"></i>
+                        </div>
+                    <h2 class="title">Programs</h2>
+                    <p>After joining the program you have to dedicate 3+ hours of learning each day.</p>
+                </div>
+              
+                <div class="card">
+                    <div class="pro">
+                        <i class="fa-solid fa-people-group"></i>
+                        </div>
+                    <h2 class="title">Workshop</h2>
+                    <p>Try creating your own work using open license</p>
+                </div>
+            </div>
+
+            <a href="#" class="promore">SEE THE WHOLE PROGRAM</a>
+            <button type="button" class="probtn">Join Microverse</button>
+        </section>

@cynthia-nwannah You are right to keep the image.

File: about.html [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+                         <h3>2017</h3>
+                        <p>internation submit in india</p>
+                     </div>
+                    </div>
+                <div class=" two">
+                    <h3>2018</h3>
+                    <p>internation submit in india</p>
+                </div>
+            </div>
+          </section>
```
Comment (elyor-doniyorov) [inline about.html]: - [ ] You must stick to the design guidelines as much as possible. I can see you have tried to implement the changes requested but it caused another design issues, please take a look at the screenshots below 👇 
- [ ] The text in images became invisible.
- [ ] As previous reviewer mentioned `See The Past Fitness Submit title has a red horizontal line. Kindly add it.
- Original design 👇 
Screenshot (279)
- Yours 👇 
Screenshot (277)

- [ ] In mobile version of this section. Kindly implement the changes requested by the previous reviewer 👇 
- [ ] Please try to stick to the design as much as possible.
- [ ] Company brands should be in 2 rows but you have them in one row. Fix it too.
- Original design 👇 
Screenshot (280)

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @cynthia-nwannah, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 9 above)
Comment (cynthia-nwannah) [reply index.html:61]: it is not compulsory nah. others have submitted successfully without putting it

**Review by zhadier — APPROVED** [review]
## Hi @cynthia-nwannah  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Tiny Note 🌤️ 
 - Please try to understand that the reviewer is trying to help you by suggesting adding a background pattern, there is nothing to gain by stopping your Project approval. Some reviewers might not ask for these changes, but some like to look out for you and help you put out the best version of your Project. As someone in the Job searching module, I can guarantee that these capstones will be some of the Projects that you would love to show to your Recruiters in your Portfolio. Bear with us, if sometimes someone asks for something extra  :)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cynthia-nwannah** [comment]
Ok thank you i understand. But how will i get the background pattern. 

Sent from Yahoo Mail on Android 
 
  On Fri, Jun 24, 2022 at 12:57 PM, Zeeshan ***@***.***> wrote:   


@zhadier approved this pull request.

Hi @cynthia-nwannah 👋
 
Your project is complete! There is nothing else to say other than... it's time to merge it 

Congratulations! 🎉
 
Keep Rocking! 👏 👏 👏
 
Tiny Note 🌤️
    
   - Please try to understand that the reviewer is trying to help you by suggesting adding a background pattern, there is nothing to gain by stopping your Project approval. Some reviewers might not ask for these changes, but some like to look out for you and help you put out the best version of your Project. As someone in the Job searching module, I can guarantee that these capstones will be some of the Projects that you would love to show to your Recruiters in your Portfolio. Bear with us, if sometimes someone asks for something extra :)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Comment by cynthia-nwannah** [comment]
And also i will also improve the work. Why i was in a hurry is bacause i am behind in schedule 

Sent from Yahoo Mail on Android 
 
  On Fri, Jun 24, 2022 at 12:57 PM, Zeeshan ***@***.***> wrote:   


@zhadier approved this pull request.

Hi @cynthia-nwannah 👋
 
Your project is complete! There is nothing else to say other than... it's time to merge it 

Congratulations! 🎉
 
Keep Rocking! 👏 👏 👏
 
Tiny Note 🌤️
    
   - Please try to understand that the reviewer is trying to help you by suggesting adding a background pattern, there is nothing to gain by stopping your Project approval. Some reviewers might not ask for these changes, but some like to look out for you and help you put out the best version of your Project. As someone in the Job searching module, I can guarantee that these capstones will be some of the Projects that you would love to show to your Recruiters in your Portfolio. Bear with us, if sometimes someone asks for something extra :)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Comment by cynthia-nwannah** [comment]
What is yiur slack name i would live to connect with yiu

Sent from Yahoo Mail on Android 
 
  On Fri, Jun 24, 2022 at 12:57 PM, Zeeshan ***@***.***> wrote:   


@zhadier approved this pull request.

Hi @cynthia-nwannah 👋
 
Your project is complete! There is nothing else to say other than... it's time to merge it 

Congratulations! 🎉
 
Keep Rocking! 👏 👏 👏
 
Tiny Note 🌤️
    
   - Please try to understand that the reviewer is trying to help you by suggesting adding a background pattern, there is nothing to gain by stopping your Project approval. Some reviewers might not ask for these changes, but some like to look out for you and help you put out the best version of your Project. As someone in the Job searching module, I can guarantee that these capstones will be some of the Projects that you would love to show to your Recruiters in your Portfolio. Bear with us, if sometimes someone asks for something extra :)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Comment by cynthia-nwannah** [comment]
The main background image is not compulsory 

Sent from Yahoo Mail on Android 
 
  On Fri, Jun 24, 2022 at 9:46 AM, ***@***.***> wrote:   


@elyor-doniyorov requested changes on this pull request.

Hi @cynthia-nwannah,

Great work on making the changes 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Suggested changes

Check the comments under the review.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

In about.html:
> @@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About page</title>
+ <link rel="stylesheet" href="about.css">
+ <script src="https://kit.fontawesome.com/2849147338.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
   
   -  You added a black navbar to only home page but if you take a look at the below screenshot of the original design you can see it is in the about page as well. Kindly add it as the previous reviewer requested 👇
   -  Also, in your about page there is still a home link. Kindly remove it instead you can use your logo as a link to your home page. 👇
   - Original design :   

   - Your design :   


In index.html:
> +        <section class="main-top">
+          <div class="background">
+            <div class="message-top">
+             <div class="message">
+                <h3>welcome to our fitness program</h3>
+                <h1>international Fitness  <br> Submit conference <br> 2022</h1>
+               <p class="fit">To be physically fit means to be in a state of health and well-being. 
+                Physical fitness is defined as the body's ability to function efficiently and effectively in work and leisure activities, to be healthy, to resist disease and to react to emergency situations.</p>
+                <p class="date">Fri 26th - Sun 28th August Bank Holiday Weekend <br>
+               <span> @raffles city convention center, singapore and more</span></p>
+             </div>
+            </div>
+          </div>
+        </section>
+        <section class="main-bottom">
+            <h2 class="section-title" id="main-program">Main Program</h2>
   
   -  You have not implemented requested change by the previous reviewer which was about background image of main program section. Kindly implement that.

In about.html:
> +            <h3>see the past fitness submit</h3>
+            <p>Take a look at the last two international fitness submit which took place at india </p>
+        </div>
+            <div id="start">
+                
+                    <div class="cont">
+                    <div class="one">    
+                         <h3>2017</h3>
+                        <p>internation submit in india</p>
+                     </div>
+                    </div>
+                <div class=" two">
+                    <h3>2018</h3>
+                    <p>internation submit in india</p>
+                </div>
+            </div>
+          </section>
   
   -    
 You must stick to the design guidelines as much as possible. I can see you have tried to implement the changes requested but it caused another design issues, please take a look at the screenshots below 👇
 
   -    
 The text in images became invisible.

   -    
 As previous reviewer mentioned `See The Past Fitness Submit title has a red horizontal line. Kindly add it.

   -    
Original design 👇   


   -    
Yours 👇   


   -    
 In mobile version of this section. Kindly implement the changes requested by the previous reviewer 👇
 
   -    
 Please try to stick to the design as much as possible.

   -    
 Company brands should be in 2 rows but you have them in one row. Fix it too.

   -    
Original design 👇

**Pull request merged**